resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -70,6 +70,15 @@ private[spark] object Config extends Logging {
       .booleanConf
       .createWithDefault(false)
 
+  val KUBERNETES_DRIVER_REUSE_PVC =
+    ConfigBuilder("spark.kubernetes.driver.reusePersistentVolumeClaim")
+      .doc("If true, driver pod tries to reuse driver-owned on-demand persistent volume claims " +
+        "of the deleted executor pods. This can be useful to reduce executor pod creation delay. " +
+        s"This requires ${KUBERNETES_DRIVER_OWN_PVC.key}=true.")
```

Comment:
Should we add more description? For example, like the description, the PVC of removed executors may not be returned immediately, so it is possibly new PVC will still be created.

Reply:
Sure. Which content do you want to see?

BTW, the main reason why it's not returned immediately is that the executor pod is in `Terminating` K8s status by K8s while the executor JVM is already dead.

Reply:
I got what is your concerns.

When you are saying `the PVC of removed executors`, you are referring the Spark executor JVM inside pod. Here, in the description, I chose `xxx of the deleted executor pods` to avoid that confusions.

For example, the pod in `Terminating` status is not deleted yet. So, it's not included at `the deleted executor pods`. So, there is no reusable PVC in this case. I'll make it more clear in the conf description.